src/libsync/configfile.h

```diff
@@ -156,8 +156,11 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     [[nodiscard]] bool optionalServerNotifications() const;
     void setOptionalServerNotifications(bool show);
 
+    [[nodiscard]] bool showChatNotifications() const;
+    void setShowChatNotifications(bool show);
+
     [[nodiscard]] bool showCallNotifications() const;
-    void setShowCallNotifications(bool show);
+    void setShowCallNotifications(const bool show);
```

Comment:
why one has `const` for the single argument and not the other one ?
method declaration with plain old data argument should be without `const` and if deemed useful, implementation could add `const`
with POD data types, `const` is not part of the method signature so `const` if needed by the implementation should not appear in the method declaration

Reply:
Thanks for the explanation, TIL

fixed now